Function floor is required, so link to libmath to avoid linking failure (underlinking)

http://bugzilla.xfce.org/show_bug.cgi?id=8230

--- Makefile.am
+++ Makefile.am
@@ -51,7 +51,8 @@
 	@SOUP_LIBS@ \
 	@LIBXEXT_LIBS@ \
 	@LIBX11_LIBS@ \
-	@XFIXES_LIBS@
+	@XFIXES_LIBS@ \
+	-lm
 
 lib_libscreenshooter_built_sources = \
 	lib/screenshooter-marshal.c lib/screenshooter-marshal.h
